### PR TITLE
fix #3

### LINKED ISSUE #3: asmara
chan

### PR DESCRIPTION
**Pull request by JoJo-Hazard**
i have change the header content

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <html>
   <head>
     <meta charset="UTF-8" />
-    <title>Dropdown menu 1</title>
+    <title>Dropdown menu new</title>
```
Comment (MerNat) [inline index.html]: @JoJo-Hazard Please change this one to `Dropdown menu another none`

**Review by MerNat — CHANGES_REQUESTED** [review]
Please change the above comments

File: index.html (hunk 1 above)
Comment (JoJo-Hazard) [reply index.html]: @MerNat   i have  made the required rectification

**Review by MerNat — APPROVED** [review]
This is approved. works perfectly

**Pull request merged**